### ISSUE TITLE
Configurable 'clear' button and canvas colour.

### ISSUE THREAD

**Issue by pandian-egnaroinc** [lovasoa/whitebophir#92]
Hi,
Continuing the discussion from https://github.com/jitsi/jitsi-meet/issues/5295

A clear button disabled by default and enabled based on a configuration variable will help self-hosted server admins to make the decision whether to enable of not. Same is the case with canvas colours to choose whiteboard vs black board etc.

I agree to the rationale of being hesitant to make it default behaviour. Neither am I asking to change the default behaviour of https://wbo.ophir.dev/

**Comment by lovasoa** [comment]
Related to #90

**Comment by jondo** [comment]
@pandian-egnaroinc, I think the configurable canvas colour should be filed as separate issue.

**Comment by pandian-egnaroinc** [comment]
> @pandian-egnaroinc, I think the configurable canvas colour should be filed as separate issue.

Sure, Will file a new issue.

Thanks,

**Comment by danionescu2007** [comment]
Hello, 

Is there a way to clear a white board, by user or by closing it, yet?
Thank you.

**Comment by lovasoa** [comment]
Hello @danionescu2007 ! No there isn't.

**Comment by danionescu2007** [comment]
Thank you for the info. Too bad 😀 there isn't.

**Comment by lovasoa** [comment]
This issue now splits into already-handled and still-open parts.

The clear button exists as the Clear tool. It is registered as `toolId: "clear"` in `client-data/tools/manifest.js`, implemented in `client-data/tools/clear/index.js`, and gated by the `clearBoard` capability so deployments can distinguish ordinary edit permission from board-wide clear permission.

The configurable canvas color / blackboard / transparent background part is still open and tracked more directly in #96.

Closing this as superseded: clear is implemented, and the remaining canvas-color work should continue in #96.
